docker/openshift/preflight/preflight.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * @file
+ * Deployment preflight checks.
+ *
+ * @see https://github.com/City-of-Helsinki/drupal-helfi-platform/blob/main/documentation/openshift.md for documentation.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Collect preflight messages.
+ *
+ * @param string $message
+ *   The message.
+ *
+ * @return array
+ *   The preflight error messages.
+ */
+function preflight_messages(string $message = '') : array {
+  static $messages = [];
+
+  if ($message) {
+    $messages[] = $message;
+  }
+  return $messages;
+}
+
+/**
+ * Mark preflight check as failed.
+ *
+ * @param string $message
+ *   The message.
+ * @param string[]|int[] $variables
+ *   The variable substitutions.
+ */
+function preflight_failed(string $message, string|int ...$variables) : void {
+  $message = sprintf('Preflight check failed: %s', sprintf($message, ...$variables));
+  preflight_messages($message);
+}
+
+/**
+ * Checks if given environment variable are set.
+ *
+ * @param array $items
+ *   The environment variables to check.
+ *
+ * @return bool
+ *   TRUE if environment variable is set.
+ */
+function environment_variable_isset(array $items) : bool {
+  foreach ($items as $item) {
+    if (getenv($item)) {
+      return TRUE;
+    }
+  }
+  return FALSE;
+}
+
+$class_loader = require_once './../vendor/autoload.php';
+include_once './sites/default/settings.php';
```

Comment:
Sonar cloud has two warnings related to this file: https://sonarcloud.io/project/issues?id=City-of-Helsinki_drupal-helfi-platform&pullRequest=197&resolved=false&types=CODE_SMELL

I think this is ok. If possible, I think the warnings should be suppressed. Does `// NOSONAR` comment work?